saagieapi/jobs/gql_queries.py

```diff
@@ -82,50 +91,102 @@
         }
     }
 }
-  """
+"""
 
 GQL_GET_JOB_INSTANCE = """
 query jobInstanceQuery($jobInstanceId: UUID!){
     jobInstance(id: $jobInstanceId){
-      id
-      number
-      status
-      startTime
-      endTime
-      jobId
-      version {
+        id
         number
-        releaseNote
-        runtimeVersion
-        commandLine
-        isMajor
-        isCurrent
-      }
+        status
+        history {
+            currentStatus {
+                status
+                details
+                reason
+            }
+        }
+        startTime
+        endTime
+        jobId
+        jobAlias
+        version {
+            number
+            releaseNote
+            runtimeVersion
+            commandLine
+            isMajor
+            isCurrent
+        }
+        executionGlobalVariablesInput{
+            key
+            value
+            isPassword
+        }
+        executionVariablesInput {
+            parentJobInstanceId
+            parentJobId
+            parentJobAlias
+            isDirectParent
+            executionVariables {
+                key
+                value
+                isPassword
+            }
+            isGlobalVariables
+        }
+        executionVariablesOutput {
+            key
+            value
+            isPassword
+        }
+        executionVariablesByKey {
+            keyVariable
+            isPassword
+            valueVariablesInputByJobInstance{
+                jobInstanceId
+                jobId
+                jobAlias
+                jobName
+                isDirectParent
+                value
+                isPassword
+            }
+            valueGlobalVariableInput
+            valueVariableOutput
+        }
     }
-  }
-  """
+}
+"""
 
 GQL_RUN_JOB = """
-  mutation runJobMutation($jobId: UUID!){
+mutation runJobMutation($jobId: UUID!){
     runJob(jobId: $jobId){
-      id
-      status
+        id
+        status
     }
-  }
-  """
+}
+"""
 
 GQL_STOP_JOB_INSTANCE = """
-  mutation stopJobInstanceMutation($jobInstanceId: UUID!){
+mutation stopJobInstanceMutation($jobInstanceId: UUID!){
     stopJobInstance(jobInstanceId: $jobInstanceId){
-      id
-      number
-      status
-      startTime
-      endTime
-      jobId
+        id
+        number
+        status
+        history {
```

Comment:
The doc for the section `Stopping a job` should be updated with the new GQL query